Project/Assets/Scripts/Utility/Logger.cs

```diff
@@ -28,10 +28,10 @@ public enum eLogLevel
     };
 
 #if STANDALONE_MONOBEHAVIOUR
-    public class Logger : MonoBehaviour
+    public class ARK_Logger : MonoBehaviour
 #else
     [Serializable]
-    public class Logger
+    public class ARK_Logger
```

Comment:
I think that c# Coding standards say that Classes should not contain _, so this should probably be ARKLogger !fix